src/components/Router/StepComponentMap.tsx

```diff
@@ -157,9 +166,46 @@ const buildCaptureStepComponents = (
         firstCaptureStepType === 'document'
       ),
     ],
+    data: [...buildDataComponents(dataStep)],
     poa: [...buildPoaComponents(mobileFlow, firstCaptureStepType === 'poa')],
     complete,
+    pass: [Pass],
+    reject: [Reject],
+  }
+}
+
+const buildDataComponents = (
```

Comment:
I think this components should not be here, but in an own file

Reply:
placed here following the similar build component as face and document and POA are also in the same place. We do you recommend we move all the buildComponents to?

Reply:
It would be good to move the big ones to their own file, maybe in other PR though.